microfrontends/app-builder-menu/src/components/hooks/useContents.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { useContent } from './useContent';
+import { renderHook } from '@testing-library/react';
+
+describe('useContent', () => {
+  it('must render a correct object', async () => {
+    const { result } = renderHook(() => useContent());
+    expect(result.current).toBeInstanceOf(Object);
+  });
+});
```

Comment:
Could you improve this test? The essence of the behaviour of `useContent` is to return the object of `content` so you can match the result with the mocked output of `getContent`